10-sharing-data.Rmd

```diff
@@ -5,6 +5,172 @@
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_201_529")
+```
+
+4. It also provides more opportunities for others to replicate your results, which could help advance not only your career, but our understanding of science and medicine.
+
+## Benefits of data sharing
+
+In addition to these benefits to yourself, data sharing has other far reaching benefits. It can help support faster advances in science and medicine, by reducing the need to collect new data, which reduces costs, time and effort, including the effort and burden that is required to collect data on or from patients.
+
+It also helps support researchers at institutes that do not have as many resources to collect data.
+
+Ultimately it can also therefore help patients benefit from research faster, as faster advances can be made through more efficient research.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Data Sharing can also help with costs related to collecting data, reduces the time and effort to collect new data, including the burden on patients, it allows research to be more efficient, the same data can be used for multiple studies, which is especially helpful if combining different kinds of data and researchers don't necessarily have to ability to collect each kind of data, it supports researchers at insitutions that have less resources, and it helps patients get the benefits of research faster.’", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_201_766")
+```
+
+## Data repositories
+
+The best way to share your data is by putting it somewhere that others can download it (and it can be kept private when necessary). There's many repositories out there that handle this for you.
+
+The repository you choose for sharing data will be highly dependent on the field you work in and the data type that you work with.
+
+Below are some of the standard repositories for data you should consider.
+
+**For a longer list of repositories, we also advise consulting this Guide on data repositories](https://www.nature.com/sdata/policies/repositories) published by Nature.**
+
+### Repositories for journal articles
+
+For manuscripts or large datasets that are of atypical format, using one of these repositories is a good idea.
+The journal you submit to may have a recommendation of one over another. If not, you might end up having a preference.
+
+- [CyVerse Data Commons Repository](https://cyverse.org/data-commons)
+- [Data Dryad](https://datadryad.org/stash)
+- [FigShare](https://help.figshare.com/article/how-to-upload-and-publish-your-data)
+- [Zenodo](https://help.zenodo.org/)
+- [GitHub](https://github.com/)
+
+### Small datasets
+
+Data sets that are small or have an atypical format can be published as supplementary files as a part of a manuscript.
```

Comment:
do you mean "and"?  just noticed the first one was large and atypical and this is small or atypical

```suggestion
Data sets that are small and have an atypical format can be published as supplementary files as a part of a manuscript.
```